### PR TITLE
New pursuit & design updates

### PR DESCRIPTION
**Pull request by sigma-andex**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -3,3 +3,15 @@ body,
...
+div[data-overlay-container=true] {
+    height: 100%;
+}
+
+#background-container {
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100vw;
+    height: 100vh
```
Comment (andys8) [inline styles.css:16]: Just a general warning that viewport sizes are - if I remember correctly - relative to the size of the device screen. Keep that in mind in regards to browser window size and special cases like navigation bar being hidden on mobile devices or a keyboard showing up. 

https://stackoverflow.com/questions/64176328/what-are-the-disadvantages-of-vh-and-vw-units

File: src/Pages/App.purs [hunk 2]
```diff
@@ -28,7 +28,7 @@ type AppProps props =
 mkApp :: forall props. Effect (EffectFn1 (AppProps props) React.JSX)
 mkApp = do
   context /\ settingsProvider <- mkSettingsProvider
-  let pursuitUrl = "https://pursuit.purescript.org"
+  let pursuitUrl = "http://localhost:8080"
```
[the remark this answers is not in the document]
Comment (sigma-andex) [reply src/Pages/App.purs:31]: yes, that is wip atm

[the remark this answers is not in the document]
Comment (sigma-andex) [reply src/Pages/App.purs:31]: using the purescript pursuit

File: styles.css (hunk 1 above)
Comment (sigma-andex) [reply styles.css:16]: yes, this class in particular is for the background. @i-am-the-slime proposed to have a separate div to render the background and give it height 100vh/vw. Seems to work. Is there a better way to do this?

File: src/Pages/App.purs (hunk 2 above)
[the remark this answers is not in the document]
Comment (sigma-andex) [reply src/Pages/App.purs:31]: I'll extract that into an env var

File: styles.css (hunk 1 above)
Comment (andys8) [reply styles.css:16]: Was a general hint. I personally would more or less use flexbox, grid or `100%`.

If it works for the background then you're probably fine with it. But might find issues. E.g. scrolling on mobile seems to show one issue. Wouldn't overthink it now, but has to work with long lists etc.


https://user-images.githubusercontent.com/13085980/214128162-c6fd47aa-0f93-4626-8ce8-7cff77ab69aa.mp4

Comment (sigma-andex) [reply styles.css:16]: yeah thats really bad. would be better to shrink this somehow than getting that scrollbar

Comment (andys8) [reply styles.css:16]: Yeah, but I was referring to the bottom. There you can see a white box appear during scrolling
 This is the effect of `100vh` I think.

Comment (sigma-andex) [reply styles.css:16]: any idea how we can fix this?

Comment (i-am-the-slime) [reply styles.css:16]: These things kill me. How do you get something that works in all circumstances without an army of testers?

Comment (andys8) [reply styles.css:16]: No. Army of testers is what you need ^^

I think fixing the navbar position, having the website container filling out the rest of the height and then only allow scrollable content within the container should work better. Right now the whole website can scroll.

Hate these things, too. I would merge it as is, fix it later, and focus on some progress with functionality. And as a general rule of thumb:  if possible, avoid any additional CSS with UI frameworks.

**Pull request merged**